example/detox/e2e/support/test_config.js

```diff
@@ -12,6 +13,33 @@ module.exports = {
         (process.env.IOS ? "http://localhost:8008" : "http://10.0.2.2:8008"),
     fileUploadSiteUrl:
         process.env.FILE_UPLOAD_SITE_URL || "http://localhost:8008",
+    secureFastImageServerClientCertUrl:
+        "https://github.com/mattermost/react-native-network-client/raw/e2e-cert/example/certs/secure_fast_image_server_client_cert.p12",
```

Comment:
I will update the secure client cert URLs later to https://github.com/mattermost/react-native-network-client/raw/master/example/certs/*_client_cert.p12 once merged to master